rosdep/python.yaml

```diff
@@ -1449,6 +1449,10 @@ python-scipy:
     utopic_python3: [python3-scipy]
     vivid: [python-scipy]
     vivid_python3: [python3-scipy]
+python-selenium:
+  ubuntu:
+    pip:
+      packages: [selenium]
```

Comment:
@wjwwood we aren't going to be debianising anything using this -- we are just using rosdep as the way to install for CI